docs/buildah-bud.md

```diff
@@ -338,9 +346,7 @@ process.
 
 **--platform**="Linux"
 
-This option has no effect on the build. Other container engines use this option
-to control the execution platform for the build (e.g., Windows, Linux) which is
-not required for Buildah as it supports only Linux.
+Specify the `os/architecture` to be used when building the image.
```

Comment:
I suggest to copy what --os and --arch state above and give an example value (e.g., "linux/amd64").

I guess we should mention for all three options which values are valid (i.e., the golang runtime values).

Reply:
> I suggest to copy what --os and --arch state above and give an example value (e.g., "linux/amd64").

Fixed that.

> I guess we should mention for all three options which values are valid (i.e., the golang runtime values).

Hm, I'm not sure since there might be incompatible os/arch combinations avilable. I think we don't have clear way to validate os and arch combinations for each go version right?

Reply:
I honestly have no idea about combinatorial issues but had the individual values for a valid arch and os in mind. The image spec, for instance, [points to the golang docs](https://github.com/opencontainers/image-spec/blob/243ea084a44451d27322fed02b682d99e2af3ba9/image-index.md#image-index-property-descriptions).